components/console-backend-service/internal/domain/application/app_resolver.go

```diff
@@ -170,9 +172,62 @@ func (r *applicationResolver) EnableApplicationMutation(ctx context.Context, app
 	return &gqlschema.ApplicationMapping{
 		Namespace:   em.Namespace,
 		Application: em.Name,
+		AllServices: allServices,
+		Services:    r.transformApplicationMappingService(em.Spec.Services),
+	}, nil
+}
+
+func (r *applicationResolver) OverloadApplicationMutation(ctx context.Context, application string, namespace string, allServices *bool, services []*gqlschema.ApplicationMappingService) (*gqlschema.ApplicationMapping, error) {
+	services = configureServices(allServices, services)
+	em, err := r.appSvc.UpdateApplicationMapping(namespace, application, services)
+	if err != nil {
+		glog.Error(errors.Wrapf(err, "while updating %s", pretty.ApplicationMapping))
+		return nil, gqlerror.New(err, pretty.ApplicationMapping, gqlerror.WithName(application), gqlerror.WithNamespace(namespace))
+	}
+
+	return &gqlschema.ApplicationMapping{
+		Namespace:   em.Namespace,
+		Application: em.Name,
+		AllServices: allServices,
+		Services:    r.transformApplicationMappingService(em.Spec.Services),
 	}, nil
 }
 
+func configureServices(allServices *bool, services []*gqlschema.ApplicationMappingService) []*gqlschema.ApplicationMappingService {
```

Comment:
You can move that function to service or just make it one of the `applicationResolver` method:
`(r *applicationResolver)`